01.fizzbuzz/fizz_buzz.rb

```diff
@@ -0,0 +1,11 @@
+[*1..20].each do |n|
```

Comment:
```suggestion
(1..20).each do |n|
```

間違いではないですが、RangeをArrayに変換する意味はないのでこれでいいかもです。

Reply:
言われてみれば…仰るとおりです！
`1..20`を()でひとつの式にグループ化してあげればよいだけで、わざわざArrayに変換する必要はありませんでした。
Commit chagesで変更をコミットしました〜！